python/cugraph/tests/test_bfs.py

```diff
@@ -13,100 +13,266 @@
 
 import gc
 from itertools import product
-import queue
-import time
 
 import numpy as np
 import pytest
-import scipy
 import cugraph
 from cugraph.tests import utils
 import rmm
+import random
 
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, this import networkx needs to be relocated in the
+# third-party group once this gets fixed.
+import warnings
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import networkx as nx
+    import networkx.algorithms.centrality.betweenness as nxacb
 
-def cugraph_call(cu_M, start_vertex):
+# =============================================================================
+# Parameters
+# =============================================================================
+RMM_MANAGED_MEMORY_OPTIONS = [False, True]
+RMM_POOL_ALLOCATOR_OPTIONS = [False, True]
 
-    G = cugraph.DiGraph()
-    G.from_cudf_edgelist(cu_M, source='0', destination='1',
-                         edge_attr='2')
+DIRECTED_GRAPH_OPTIONS = [True]
```

Comment:
Most of these graphs are actually undirected. I think it would make more sense to store the undirected versions and let cuGraph load a `Graph` instead of storing them as directed graphs and loading them in `DiGraph`. This is not related to this PR in particular but something we can address separately for all the test suite. @Iroy30, thoughts?

Reply:
https://github.com/rapidsai/cugraph/issues/872